CIRG-PEG.json

```diff
@@ -4,7 +4,7 @@
   "name": "Pain intensity, Enjoyment of life, General activity (PEG) 3 item pain scale",
   "title": "Pain intensity, Enjoyment of life, General activity (PEG) 3 item pain scale",
   "status": "draft",
-  "description": "Pain intensity, Enjoyment of life, General activity (PEG) 3 item pain scale. Based primarily on https://loinc.org/91148-7/ and to a lesser extent https://fhir.loinc.org/Questionnaire/?url=http://loinc.org/q/91148-7, using codes from the former. Note that these two references had different codes for the first question and that it was implemented at fhir.loinc.org as a decimal (not a choice), but academic references did not support that. The CIRG-PEG-SUM item is a scoring variant used at UW Medicine's Center for Pain Relief.",
+  "description": "Pain intensity, Enjoyment of life, General activity (PEG) 3 item pain scale. Reference: [https://pubmed.ncbi.nlm.nih.gov/19418100/](https://pubmed.ncbi.nlm.nih.gov/19418100/). The CIRG-PEG-SUM item is a scoring variant used at UW Medicine's Center for Pain Relief. <-- This FHIR resource is based primarily on https://loinc.org/91148-7/ and to a lesser extent https://fhir.loinc.org/Questionnaire/?url=http://loinc.org/q/91148-7, using codes from the former. Note that these two references had different codes for the first question and that it was implemented at fhir.loinc.org as a decimal (not a choice), but academic references did not support that. -->",
```

Comment:
@mcjustin It seems that the comment is still rendered:  
![Screen Shot 2023-10-12 at 4 15 46 PM](https://github.com/uwcirg/fhir-questionnaires/assets/12942714/9bfecbdd-d8a7-4bc6-88d2-4637f591f769)

Reply:
are you using showdown.js for that, or?

Reply:
@mcjustin I use this [commonmark library](https://github.com/commonmark/commonmark.js/). I wasn't able to use showdown.js because it requires higher version of Node, which is incompatible with the current Node used by COSRI frontend.

Reply:
@achen2401  sorry, I was using bad html comment syntax there (`<--` instead of `<!--`) - I've fixed that now can you please try again? It looks like commonmark.js should parse this as expected. https://spec.commonmark.org/dingus/

Reply:
@mcjustin thank you.  this looks good now.
![Screen Shot 2023-10-13 at 9 06 09 AM](https://github.com/uwcirg/fhir-questionnaires/assets/12942714/4c86e34b-c85b-4bb4-9e1b-68f497e67184)